src/components/Keg.vue

```diff
@@ -110,14 +140,15 @@ export default {
 }
 
 function perc2color(perc) {
-	var r, g, b = 0;
```

Comment:
in the browserlist config in package.json, we are requiring at least the latest 2 versions of each browser in order to run this, so we should be able to assume that let/const are available and i would switch to using those instead.
```
let r;
let g;
const b = 55;
...
const h = ...;
```